---
layout: page
title: Research
permalink: /research/
---

My background is phylogenetic methods and phyloinformatics. How do we build 
evolutionary trees from sequence data and how do we synthesize and share 
information about the tree of life? More broadly, I am interested in the 
standards, tools, training, and communities that we need in order to make 
biodiversity data easily and widely available. 

## Peer-reviewed publications

McTavish, E. J., Drew, B. T., Redelings, B., & Cranston, K. A. (2017). How and Why to Build a Unified Tree of Life. BioEssays, 39(11). https://dx.doi.org/10.1002/bies.201700114

Stoltzfus, A., Rosenberg, M., Lapp, H., Budd, A., Cranston, K., Pontelli, E., ... & Vos, R. A. (2017). Community and Code: Nine Lessons from Nine NESCent Hackathons. F1000Research, 6. http://dx.doi.org/10.12688/f1000research.11429.1

Wilson, G., Bryan, J., Cranston, K., Kitzes, J., Nederbragt, L., \& Teal, T. K. (2017). Good enough practices in scientific computing. PLoS Comput Biol 13(6): e1005510. [doi:10.1371/journal.pcbi.1005510](https://doi.org/10.1371/journal.pcbi.1005510)

Rees, J., & Cranston, K. (2017). Automated assembly of a reference taxonomy for phylogenetic data synthesis. Biodiversity Data Journal, 5, e12581. [doi:10.3897/BDJ.5.e12581](https://doi.org/10.3897/BDJ.5.e12581)

McTavish, E. J., Hinchliff, C. E., Allman, J. F., Brown, J. W., Cranston, K. A., Holder, M. T., ... & Smith, S. A. (2015). Phylesystem: a git-based data store for community curated phylogenetic estimates. Bioinformatics, btv276. [doi:10.1093/bioinformatics/btv276](http://dx.doi.org/10.1093/bioinformatics/btv276).

Hinchliff, C. E., Smith, S. A., Allman, J. F., Burleigh, J. G., Chaudhary, R., Coghill, L. M., ... & Cranston, K. A. (2015). Synthesis of phylogeny and taxonomy into a comprehensive tree of life. Proceedings of the National Academy of Sciences, 112(41), 12764-12769. [doi:10.1073/pnas.1423041112](http://dx.doi.org/10.1073/pnas.1423041112). 

Ksepka, D.T., Parham, J.F., Allman, J.F., Benton, M.J., Carrano, M.T., Cranston, K.A., ... & Warnock, R.C.M. (2015) The Fossil Calibration Database, A New Resource for Divergence Dating. Systematic Biology. [doi:10.1093/sysbio/syv025](http://dx.doi.org/10.1093/sysbio/syv025)

Teal, T. K., Cranston, K. A., Lapp, H., White, E., Wilson, G., Ram, K., Pawlik, A. (2015) Data Carpentry: Workshops to Increase Data Literacy for Researchers. International Journal of Digital Curation. 10(1):135-143. [doi:10.2218/ijdc.v10i1.351](http://dx.doi.org/10.2218/ijdc.v10i1.351)

Cranston, K. A., Harmon, L. J., O'Leary, M. A., Lisle, C. (2014) Best Practices for Data Sharing in Phylogenetic Research. PLOS Currents Tree of Life. Jun 19. Edition 1. [doi:10.1371/currents.tol.bf01eff4a6b60ca4825c69293dc59645](http://dx.doi.org/10.1371/currents.tol.bf01eff4a6b60ca4825c69293dc59645)

Stoltzfus, A., Lapp, H., Matasci, N., Deus, H., Sidlauskas, B., Zmasek, C. M., ... & Jordan, G. (2013). Phylotastic! Making tree-of-life knowledge accessible, reusable and convenient. BMC bioinformatics, 14(1), 158. [doi:10.1186/1471-2105-14-158](http://dx.doi.org/10.1186/1471-2105-14-158)

Rodrigo, A., Alberts, S., Cranston, K., Kingsolver, J., Lapp, H., McClain, C., ... & Wiegmann, B. (2013). Science incubators: synthesis centers and their role in the research ecosystem. PloS biology, 11(1), e1001468. [doi:10.1371/journal.pbio.1001468](http://dx.doi.org/10.1371/journal.pbio.1001468)

Goff, S. A., Vaughn, M., McKay, S., Lyons, E., Stapleton, A. E., Gessler, D., ... & Kleibenstein, D. J. (2011). The iPlant collaborative: cyberinfrastructure for plant biology. Frontiers in plant science, 2. [doi:10.3389/fpls.2011.0003](http://dx.doi.org/10.3389/fpls.2011.0003)

Evans, M. E., Hearn, D. J., Theiss, K. E., Cranston, K., Holsinger, K. E., & Donoghue, M. J. (2011). Extreme environments select for reproductive assurance: evidence from evening primroses (Oenothera). New Phytologist, 191(2), 555-563. [doi:10.1111/j.1469-8137.2011.03697.x](http://dx.doi.org/10.1111/j.1469-8137.2011.03697.x)

Ammiraju, J. S., Fan, C., Yu, Y., Song, X., Cranston, K. A., Pontaroli, A. C., ... & Wing, R. A. (2010). Spatiotemporal patterns of genome evolution in allotetraploid species of the genus Oryza. The Plant Journal, 63(3), 430-442. [doi:10.1111/j.1365-313X.2010.04251.x](http://dx.doi.org/10.1111/j.1365-313X.2010.04251.x)

Cranston, K. A. (2010) Quantifying gene tree incongruence across varying phylogenetic depths. In L. Lacey Knowles, Laura S. Kubatko (Eds.) Estimating Species Trees: Practical and Theoretical Aspects, Wiley-Blackwell.

Cranston, K. A., Hurwitz, B., Sanderson, M. J., Ware, D., Wing, R. A., & Stein, L. (2010). Phylogenomic analysis of BAC-end sequence libraries in Oryza (Poaceae). Systematic botany, 35(3), 512-523. [doi:10.1600/036364410792495872](http://dx.doi.org/10.1600/036364410792495872)

Cranston, K. A., Hurwitz, B., Ware, D., Stein, L., & Wing, R. A. (2009). Species trees from highly incongruent gene trees in rice.  Systematic Biology 58(5), 489-500. [doi:10.1093/sysbio/syp054](http://dx.doi.org/10.1093/sysbio/syp054)

Sanderson, M. J., Boss, D., Chen, D., Cranston, K. A., & Wehe, A. (2008). The PhyLoTA Browser: processing GenBank for molecular phylogenetics research. Systematic Biology 57(3), pp. 335-346. [doi:10.1080/10635150802158688](http://dx.doi.org/10.1080/10635150802158688)

Cranston, K. A., & Rannala, B. (2007). Summarizing a posterior distribution of trees using agreement subtrees. Systematic Biology 56(4), pp. 578-590.
[doi:10.1080/10635150701485091](http://dx.doi.org/10.1080/10635150701485091)

Cranston, K., & Rannala, B. (2005). Closing the gap between rocks and clocks. Heredity, 94(5), 461-462. [doi:10.1038/sj.hdy.6800644](http://dx.doi.org/10.1038/sj.hdy.6800644)

## Pre-prints and other non-peer-reviewed

(published preprints not shown)

Katz, D.S., Choi, S.T., Wilkins-Diehr, N., Hong, N.C., Venters, C.C., ... & Littauer, R. Report on the Second Workshop on Sustainable Software for Science: Practice and Experiences (WSSSPE2). [arXiv:1507.01715](http://arxiv.org/abs/1507.01715)

Katz, D. S., Allen, G., Hong, N. C., Cranston, K., Parashar, M., Proctor, D., ... & Wilkins-Diehr, N. (2014). Second Workshop on on Sustainable Software for Science: Practice and Experiences (WSSSPE2): Submission, Peer-Review and Sorting Process, and Results.  [arXiv:1411.3464](http://arxiv.org/abs/1411.3464).

Vision, T., & Cranston, K. A. (2014) Open data for evolutionary synthesis: an introduction to the NESCent collection. Nature Scientific Data. [doi:10.1038/sdata.2014.30](http://dx.doi.org/10.1038/sdata.2014.30)

Cranston, K. A., Blackburn, D., Brown, J. W., Dececchi, A., Gardner, N., Greshake, B., ... & Wolfe, J. (2014): Simple rules for sharing phylogenetic data. figshare. [doi:10.6084/m9.figshare.997763](http://dx.doi.org/10.6084/m9.figshare.997763)

Rees, J. A., Cranston, K. A., Lapp, H., & Vision, T. (2013): Response to GBIF request for consultation on data licenses. figshare. [doi:10.6084/m9.figshare.799766](http://dx.doi.org/10.6084/m9.figshare.799766)

Cranston, K. A., Vision, T., O'Meara, B., & Lapp, H. (2013): A grassroots approach to software sustainability. figshare. [doi:10.6084/m9.figshare.790739](http://dx.doi.org/10.6084/m9.figshare.790739)
